demo/README.md

```diff
@@ -0,0 +1,7 @@
+# Uast viewer example
+
+Start the demo:
+
+```bash
+$ yarn start
```

Comment:
This is kind of basic, but for the sake of completeness I'd put that first you need to run `yarn install`.
Also, title case in the title.

Reply:
fixed. but readme will be updated later when lib become more mature as well as demo